packages/coordinator/src/screens/bookedDonation/BookedAppointmentScreen.tsx

```diff
@@ -72,13 +75,13 @@ export default function BookedAppointmentScreen(
   );
 }
 
-function nameBar(name: string) {
+function NameBar({ ...props }) {
```

Comment:
Don't you have to specify the type?
I think you need to change it to `NameBar(props: BookedAppointmentScreenProps) {`